src/sstc.adoc

```diff
@@ -141,47 +141,47 @@ ensures compatibility with existing guest VS-mode software that uses this SEE
 facility, while new VS-mode software takes advantage of vstimecmp directly.)
 ====
 
-==== Hypervisor Interrupt Registers (hvip, hip, and hie)
+==== Hypervisor Interrupt (`hvip`, `hip`, and `hie`) Registers 
 
 This extension modifies the description of the VSTIP/VSTIE bits in the hip/hie
 registers as follows:
 
 Bits hip.VSTIP and hie.VSTIE are the interrupt-pending and interrupt-enable
 bits for VS-level timer interrupts. VSTIP is read-only in hip, and is the
 logical-OR of hvip.VSTIP and the timer interrupt signal resulting from
-vstimecmp (if vstimecmp is implemented). The hip.VSTIP bit, in response to
-timer interrupts generated by vstimecmp, is set and cleared by writing
-vstimecmp with a value that respectively is less than or equal to, or greater
-than, the current (time + htimedelta) value. The hip.VSTIP bit remains defined
+vstimecmp (if `vstimecmp` is implemented). The hip.VSTIP bit, in response to
+timer interrupts generated by `vstimecmp`, is set and cleared by writing
+`vstimecmp` with a value that respectively is less than or equal to, or greater
+than, the current (`time` + `htimedelta`) value. The hip.VSTIP bit remains defined
 while V=0 as well as V=1.
 
-==== Hypervisor Counter-Enable Register (hcounteren)
+==== Hypervisor Counter-Enable (`hcounteren`) Register 
 
 This extension adds to the description of the TM bit in this register as
 follows:
 
-In addition, when the TM bit in the hcounteren register is clear, attempts to
-access the vstimecmp register (via stimecmp) while executing in VS-mode will
-cause a virtual instruction exception if the same bit in mcounteren is set.
-When this bit and the same bit in mcounteren are both set, access to the
-vstimecmp register (if implemented) is permitted in VS-mode.
+In addition, when the TM bit in the `hcounteren` register is clear, attempts to
+access the `vstimecmp` register (via stimecmp) while executing in VS-mode will
+cause a virtual instruction exception if the same bit in `mcounteren` is set.
+When this bit and the same bit in `mcounteren` are both set, access to the
+`vstimecmp` register (if implemented) is permitted in VS-mode.
 
-=== Environment Config (menvcfg/henvcfg) Support
+=== Environment Config (`menvcfg` or `henvcfg`) Support
```

Comment:
"and" instead of "or"?  (I know "or" is the literal replacement for the slash, but "and" is clearer.)